omero/sysadmins/unix/install-web/walkthrough/omeroweb-install-with-server-centos7-ice3.6.rst

```diff
@@ -18,12 +18,6 @@ Installing prerequisites
 
 **The following steps are run as root.**
 
-Install Pillow and Numpy:
```

Comment:
Why are some blocks removed?

Reply:
b/c they are already installed in the server page and this is the doc for web with server